docs/api/auxiliary.rst

```diff
@@ -1,6 +1,11 @@
+.. meta::
+  :description: hipSOLVER documentation and API reference library
+  :keywords: hipSOLVER, rocSOLVER, ROCm, API, documentation
+
+.. _library_auxiliary:
 
 ***************************
-LAPACK Auxiliary Functions
+LAPACK auxiliary functions
```

Comment:
Shouldn't this be "hipsolver LAPACK auxiliary functions"?

Reply:
Updated